src/controller/index.py

```diff
@@ -1,12 +1,40 @@
-from fastapi import APIRouter
-from fastapi.responses import FileResponse
-
-from ..useful.project import get_file_path
+from fastapi import APIRouter, Request
+from fastapi.responses import HTMLResponse
+from fastapi.templating import Jinja2Templates
+from jinja2 import Environment, FileSystemLoader
+from ..api import product as product_api
+from ..domain import product as product_domain
 
 router = APIRouter()
 
-@router.get("/")
-async def index():
-    navbar_path = "../assets/templates/base.html"
-    path = get_file_path(navbar_path)
-    return FileResponse(path)
\ No newline at end of file
+class ExtendedEnvironment(Environment):
```

Comment:
Please take a look, what is this used for?

Reply:
In the constructor (__init__), the int function is added to the global variables of the Jinja2 environment. This allows the int function to be used directly in Jinja2 templates in index.html without the need to explicitly import it into each template.

Reply:
Ok